kratos/spaces/ublas_space.h

```diff
@@ -538,6 +538,10 @@ class UblasSpace
         return inner_prod(row(rA, i), rX);
     }
 
+    static void SetLocalValue(VectorType& rX, IndexType i, double value)
```

Comment:
I understand the reason for this function being called `SetLocalValue`. But I do not like it here in ublas_space as no other function name in here indicates distinction between local and global. Also this is because I think there is no need. 

Why not modify `SetValue` itself with a default flag (`check_blobal=false`) which does nothing in `ublas_space` but will be used in `trilinos_space` ?

Reply:
that would be an extra if...